packages/ibm-products/src/components/Datagrid/Datagrid/DatagridRow.tsx

```diff
@@ -213,6 +225,10 @@ const DatagridRow = (datagridState: DataGridState) => {
                     associatedHeader &&
                     associatedHeader.length &&
                     isValidElement(associatedHeader[0]?.slug),
```

Comment:
these and above line can be removed, as we apply corresponding aiLabel classname conditionally on presence of aiLable or slug prop

Reply:
Since some adapters are already using the existing slug classes, completely changing the classnames would break their code. Hence a separate class is added for AiLabel. Once we stop the support for slug completely, we could just remove the slug classes. For now it would be safer to keep separate classes.